packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -82,46 +88,77 @@ function getListboxOptionsFromL1Network(network: Chain) {
 
 function NetworkContainer({
   network,
+  balanceFor,
   children
 }: {
   network: Chain
+  balanceFor?: string
```

Comment:
Looks like we're just passing an address here, so might as well call it `customAddress`? Goes well with the `customAddressBannerClassName`